cinema/views.py

```diff
@@ -173,3 +130,29 @@ def get_serializer_class(self):
 
     def perform_create(self, serializer):
         serializer.save(user=self.request.user)
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer  # Assuming MovieSerializer exists
```

Comment:
Ensure that the `MovieSerializer` is correctly defined in the serializers module. This comment serves as a reminder to verify its existence and correctness.